net/wgpeerselector/README.md

```diff
@@ -0,0 +1,97 @@
+wgpeerselector
+==============
+
+The wireguard peer selector is a daemon allowing to connect to only one
+wireguard peer of a set of N peers. It does that by randomingly picking one
+peer, trying to connect and on failure trying to connect to the next peer.
+It is comparable to setting the "peer limit" to one in fastd. The daemon
+is written in lua and hardwired to OpenWrt mechanisms.
+
+Features:
+- Status socket via UBUS.
+- Routes for "Allowed IPs" are installed to the kernel.
+- Synchronize time via NTP.
+    - Currently necessary in wireguard to reestablish a connection after
+      rebooting. [Reference](https://lists.zx2c4.com/pipermail/wireguard/2019-February/003850.html)
```

Comment:
The only _real_ requirement is that it's monotonic. So you could just store the time to flash at every N interval, restore from that at boot, and have at most N interval of downtime while the clock advances to wherever it was when power was cut.

Reply:
Of course it would also be possible to write the time to flash. Since neither Gluon nor OpenWrt currently do that, the only option for this package is to synchronize the time via ntp. I think, as we have attached the link to the "Reference" here, this is sufficient...